Branch-SDK/Branch-SDK/BNCPreferenceHelper.m

```diff
@@ -542,6 +540,57 @@ - (NSInteger)getBranchViewCount:(NSString *)branchViewID {
     return count;
 }
 
+- (void)saveBranchAnalyticsData:(NSDictionary *)analyticsData {
+    if (_sessionID != nil) {
+        if (_savedAnalyticsData == nil) {
+            _savedAnalyticsData = [self getBranchAnalyticsData];
+        }
+        NSMutableArray *viewDataArray;
+        if ([_savedAnalyticsData objectForKey:_sessionID] != nil) {
```

Comment:
just a bit cleaner to do `if (![_savedAnalyticsData objectForKey:_sessionID]) {`